shared/tracker/proofs.render.desktop.js

```diff
@@ -77,10 +77,14 @@ export class ProofsRender2 extends Component {
         color = this._isTracked(proof) ? globalColorsDZ2.green : globalColorsDZ2.blue
         break
       }
-      case proofChecking:color = color = '#999'; break
-      case proofRevoked: color = globalColorsDZ2.red; break
-      case proofWarning: color = globalColorsDZ2.orange; break
-      case proofError: color = globalColorsDZ2.red; break
+      case proofChecking:
+        color = color = '#999'
```

Comment:
we should use black10 or black20 for this
@cecileboucheron in zeplin this is actually black15, a non-standard color.